spec/requests/api/v1.0/portfolio_items_spec.rb

```diff
@@ -163,28 +163,50 @@
 
   context "when adding portfolio items" do
     let(:add_to_portfolio_svc) { double(ServiceOffering::AddToPortfolioItem) }
-    let(:params) { { :service_offering_ref => service_offering_ref } }
-    let(:permitted_params) { ActionController::Parameters.new(params).permit(:service_offering_ref) }
+    let(:params) { {:service_offering_ref => service_offering_ref, :portfolio_id => portfolio.id} }
+    let(:permitted_params) { ActionController::Parameters.new(params).permit(:service_offering_ref, :portfolio_id) }
 
     before do
       allow(ServiceOffering::AddToPortfolioItem).to receive(:new).with(permitted_params).and_return(add_to_portfolio_svc)
     end
 
-    it "returns not found when topology doesn't have the service_offering_ref" do
-      allow(add_to_portfolio_svc).to receive(:process).and_raise(topo_ex)
+    context "when a portfolio_id is not included in the parameters" do
+      let(:params) { {:service_offering_ref => service_offering_ref} }
 
-      post "#{api_version}/portfolio_items", :params => params, :headers => default_headers
-      expect(response).to have_http_status(:service_unavailable)
+      it "returns a 400" do
+        post "#{api_version}/portfolio_items", :params => params, :headers => default_headers
+        expect(response).to have_http_status(:bad_request)
+      end
+
+      it "returns a required parameter error in the body" do
+        post "#{api_version}/portfolio_items", :params => params, :headers => default_headers
+        expect(json["errors"].first["detail"]).to match(/missing required parameters: portfolio_id/)
```

Comment:
@eclarizio we have helper for this

https://github.com/RedHatInsights/catalog-api/blob/7f7048693df647b0b9dc336d755e607dc1337505/spec/support/requests_spec_helper.rb#L14